security/medusa/l2/kobject_fuck.h

```diff
@@ -9,8 +9,7 @@
 #include <linux/errno.h>
 #include <linux/medusa/l3/registry.h>
 #include <linux/medusa/l1/inode.h>
-
-#define inode_security(inode) (*(struct medusa_l1_inode_s*)(inode->i_security))
```

Comment:
This definition of inode_security is tricky. Actually the same macro is defined in kobject_file.h